pkg/odo/cli/create/namespace/namespace.go

```diff
@@ -99,10 +98,6 @@ func (nco *NamespaceCreateOptions) Run(ctx context.Context) (err error) {
 	}
 	s.End(true)
```

Comment:
Can you move this `s.End(true)` line after the `for` loop? Otherwise, it looks like `odo` is blocked if the `for` loop  takes too long to return.

[Screencast from 2023-06-27 09-43-07.webm](https://github.com/redhat-developer/odo/assets/593208/763f8cd2-79fe-4df4-834e-19f433250823)